active/operators/trashman/check_resynth_tasks.py

```diff
@@ -1,9 +1,56 @@
 import json
 from datetime import datetime, timedelta
-"""
-TODO: Implement redriving capabilities.
 
-"""
-def check_resynth_tasks(**kwargs):
-    (run_id, task_info) = kwargs['ti'].xcom_pull(task_ids='generate_job_id')
-    return None
\ No newline at end of file
+import utilities.common_variables as common_variables
+
+from utilities.common_hooks import AIRFLOW_NLP_DB
+
+'''
+TODO: 
+this clause DOES NOT take into account (hdcorcablobid,hdcpupdatedate) jobs which failed and then later succeeded.
+Need to account for this somehow
+'''
+REDRIVEABLE_CLAUSE = ("resynth_status = {scheduled_var} AND DATEDIFF(DAY, annotation_creation_date, GetDate()) >= {delta} "
+                     " OR resynth_status = {failure_var} ".format(scheduled_var=common_variables.JOB_RUNNING,
+                                   delta=common_variables.common_variables,
+                                   failure_var=common_variables.JOB_FAILURE))
+
+SQL_FETCH_AF3_REDRIVABLE_JOBS = ("SELECT af3_runs_id"
```

Comment:
should we keep this closer to the naming convention we have for other sql? like target_select/source_select ?

Reply:
or if you think this is different enough that it's it's own convention...

Reply:
maybe? I don't really like that naming convention to begin with, tbh. it doesn't tell you much about the contents of the statement.